File: code-ninja-course/CSHARP/CleanCode/Code/SOLIDSolution/Students/StudentRepository.cs
namespace CleanCode.SOLIDSolution
{
  public class StudentRepository
  {
  	public void add(Student student)
    {
  		throw new NotImplemented();
  	}

    public bool exists(string emailAddress) {
  		throw new NotImplemented();
  	}

  	public Student createStudent(string emailAddress, int universityID)
    {
  		  return new Student(emailAddress, universityID);
  	}
  }
}

File: code-ninja-course/JAVA/src/main/java/com/codeninja/solidsolution/students/StudentRepository.java
package com.codeninja.solidsolution.students;

import com.codeninja.solidsolution.exceptions.*;

	public class StudentRepository {
		public void add(Student student) {
			throw new NotImplemented();
		}
		public boolean exists(String emailAddress) {
			throw new NotImplemented();
		}
	
		public Student createStudent(String emailAddress, int universityID) {
			return new Student(emailAddress, universityID);
		}

	
	

}
